include/dlr.h

```diff
@@ -12,6 +12,17 @@ extern "C" { // Open extern "C" block
 #endif
 #endif // __cplusplus
 
+/*! \brief major version */
+#define DLR_MAJOR 1
+/*! \brief minor version */
+#define DLR_MINOR 0
+/*! \brief patch version */
+#define DLR_PATCH 0
+/*! \brief DLR version */
+#define DLR_VERSION (DLR_MAJOR*10000 + DLR_MINOR*100 + DLR_PATCH)
```

Comment:
Do we need this and the one below? If so, please change `GetDLRVersion` to reflect how you get the version `1.x.y`. Otherwise, we can just remove.

Reply:
GetDLRVersion() outputs "1.x.y" string via 'out' arg now.